### PR TITLE
Separate pending org applications and add banner

### PR DESCRIPTION
**Pull request by di**

### CODE REVIEW

**Comment by di** [comment]
Looks like this:

image

File: warehouse/templates/manage/organizations.html [hunk 1]
```diff
@@ -132,6 +132,19 @@ <h3 class="organization-snippet__title">
         <p>{% trans %}You have not joined any organizations on PyPI, yet.{% endtrans %}</p>
       </div>
     {% endfor %}
+  </div>
+
+  {% if organization_applications|selectattr('is_approved', 'equalto', None)|list|length > 0 %}
```
Comment (miketheman) [inline warehouse/templates/manage/organizations.html:137]: **question:** ‏is this attempting to to perform the equivalent of `if orgapps.is_approved is None`? 
If so, the jinja of `{% if organization.is_approved is none ... %}` ought to work.

I hit this before, and this helped: https://jinja.palletsprojects.com/en/3.1.x/templates/#jinja-tests.none:~:text=The%20special%20constants,the%20lowercase%20versions

**Review by miketheman — APPROVED** [review]
Generally fine if it works, a suggestion for improvement vs selectattr if that's what's intended.

File: warehouse/templates/manage/organizations.html (hunk 1 above)
Comment (di) [reply warehouse/templates/manage/organizations.html:137]: It's checking if there are _any_ organizations for which `is_approved` is `None`, not sure your example would work there?

I'll change the casing to `none` for consistency though:

```suggestion
  {% if organization_applications|selectattr('is_approved', 'equalto', none)|list|length > 0 %}
```

**Pull request merged**